### DISCUSSION TITLE
Data format discussion

### DISCUSSION THREAD

**Discussion by thenishantsapkota** [Know-Nepal/tech-companies#1] [Announcements]
Hello everyone,

In this repository, we'll be listing each and every Tech Company based in Nepal.
Currently we have thought of  this data format.
```json
{
 "name": "Awesome Company",
 "establishedDate": "2022-11-01",
 "location": "Fully remote"
}
```

What are the other attributes that we can include?
Feel free to comment down below.

**Comment by bsantosh909** [comment]
Hi @thenishantsapkota 👋 

We can possibly add more information like:
- Website (Every tech company should have it)
- Github (Optional, can be empty string if not available)

**Reply by thenishantsapkota** [reply]
Nice.
I completely forgot about these two attributes.
Maybe we could have Social Media links as well?
Like `Facebook` and `Instagram`

**Reply by bsantosh909** [reply]
Ahh right,  social media handlers are one of the most important thing for today's era.
We can have social handlers like `Facebook`, `Instagram`, `Youtube`, `LinkedIn`, `Twitter` (_Please add any more that can be added_)

Maybe we can even have a `social` key at the top level of an organization object and have all these values inside??

**Reply by bsantosh909** [reply]
NOTE: We can even add the items on top level itself.

**Reply by thenishantsapkota** [reply]
```json
{
 "name": "Awesome Company",
 "establishedDate": "2022-11-01",
 "location": "Fully remote",
 "socials": {
   "facebook": "https://facebook.com",
   "instagram": "https://instagram.com",
   }
}
```

We could do something like this.

**Reply by bsantosh909** [reply]
Sounds good to me 🙇

**Comment by devsargam** [comment]
@thenishantsapkota what about using [unix timestamp](https://www.unixtimestamp.com/) instead of "year-month-day" string format.
This will provide a universal way of parsing time in all languages.

**Reply by thenishantsapkota** [reply]
That could be used as well.
We wouldn't know the exact time of the company creation but we could use 00:00 for that.
Nice idea.

**Reply by bsantosh909** [reply]
Sounds like an great idea indeed
